src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -401,25 +420,18 @@ internal sealed partial class AssignmentExpression : BaseAssignmentExpression, I
         /// Value to be assigned to the target of the assignment.
         /// </summary>
         public override IOperation Value { get; }
-        public override void Accept(OperationVisitor visitor)
-        {
-            visitor.VisitAssignmentExpression(this);
-        }
-        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
-        {
-            return visitor.VisitAssignmentExpression(this, argument);
-        }
     }
 
     /// <summary>
     /// Represents an assignment expression.
     /// </summary>
-    internal sealed partial class LazyAssignmentExpression : BaseAssignmentExpression, IAssignmentExpression
+    internal sealed partial class LazySimpleAssignmentExpression : BaseSimpleAssignmentExpression, ISimpleAssignmentExpression
     {
         private readonly Lazy<IOperation> _lazyTarget;
```

Comment:
As a general question about the lazy pattern in IOperations, why not just implement this as: 
`class Lazy { private readonly Lazy<SimpleAssignmentExpression>; ... }`

Do we expect that the different lazy parts with materialized separately?
[not blocking this PR]